modules/shell/command.go

```diff
@@ -50,6 +52,35 @@ func RunCommandAndGetOutput(t *testing.T, command Command) string {
 // RunCommandAndGetOutputE runs a shell command and returns its stdout and stderr as a string. The stdout and stderr of that command will also
 // be printed to the stdout and stderr of this Go program to make debugging easier.
 func RunCommandAndGetOutputE(t *testing.T, command Command) (string, error) {
+	allOutput := []string{}
+	err := runCommandAndStoreOutputE(t, command, &allOutput, &allOutput)
+
+	output := strings.Join(allOutput, "\n")
+	return output, err
+}
+
+// RunCommandAndGetStdOut runs a shell command and returns solely its stdout (but not stderr) as a string. The stdout
+// and stderr of that command will also be printed to the stdout and stderr of this Go program to make debugging easier.
+func RunCommandAndGetStdOut(t *testing.T, command Command) string {
+	output, err := RunCommandAndGetStdOutE(t, command)
+	require.NoError(t, err)
+	return output
+}
+
+// RunCommandAndGetStdOutE runs a shell command and returns solely its stdout (but not stderr) as a string. The stdout
+// and stderr of that command will also be printed to the stdout and stderr of this Go program to make debugging easier.
+func RunCommandAndGetStdOutE(t *testing.T, command Command) (string, error) {
+	stdout := []string{}
+	stderr := []string{}
+	err := runCommandAndStoreOutputE(t, command, &stdout, &stderr)
+
+	output := strings.Join(stdout, "\n")
+	return output, err
+}
+
+// RunCommandAndGetOutputE runs a shell command and returns its stdout and stderr as a string. The stdout and stderr of that command will also
+// be printed to the stdout and stderr of this Go program to make debugging easier.
```

Comment:
NIT: This needs to be updated to reflect the new function.

Reply:
https://github.com/gruntwork-io/terratest/pull/324/commits/3a650a70078fefc3af67ab2f35e1052acc15c12e